explore/src/clue/scala/queries/common/GroupQueriesGQL.scala

```diff
@@ -22,6 +22,7 @@ object GroupQueriesGQL:
         observation {
           id
           groupIndex
+          existence
```

Comment:
I'm not sure if there's any way to query to already filter deleted observations out, but that might be an even better solution

Reply:
Right, we would get less data in the first place, @swalker2m ?